dev-libs/utfcpp/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>arfrever.fta@gmail.com</email>
+		<name>Arfrever Frehtes Taifersar Arahesis</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>cjk@gentoo.org</email>
```

Comment:
guru packages should not have official projects listed as maintainers

Reply:
I just copied the ebuild of the official tree, therefore leaving the maintainers as is. This is now fixed (I removed all maintainers).